.storybook/preview.js

```diff
@@ -54,16 +54,18 @@ export const decorators = [
       default:
       case 'default':
         return (
-          <Theme.Provider theme={theme} disableAnimations={isChromatic()}>
+          <RenderPerformanceProfiler id={context.id} kind={context.kind} view="default">
```

Comment:
Do we expect the view to affect the components' perf?

Reply:
Potentially yeah. The mount time will be different if you just displaying the story, or if you're displaying a story side by side. It's a duplicate effectively